code/headers/can_bus_pending_task.hpp

```diff
@@ -3,13 +3,18 @@
 #include <module_task_base.hpp>
 
 namespace r2d2::module_scheduler {
+    // this task checks the communication bus for every module and unlocks the
+    // task when there are frames ready to be processed
     template <size_t Size>
     class can_bus_pending_task_c : public rtos::task<256> {
     private:
         std::array<module_task_base_c *, Size> tasks;
         size_t next_task_index = 0;
 
     public:
+        // adds a module to the list of tasks
+        // this task will loop thru this list to check for frames for every
```

Comment:
thru should be through

Reply:
https://github.com/R2D2-2019/module_scheduler/pull/5/commits/fbc15c836927b0f79866df015a2da0bd959d0550